content/docs/installation.md

```diff
@@ -183,7 +183,7 @@ service miniflux start
 
 **Docker Registries:**
 
-- Docker Hub: `docker.io/miniflux/miniflux`
+- Docker Hub: `hub.docker.com/r/miniflux/miniflux`
```

Comment:
ahhh gotcha! Since the github url does redirect to the information page, I incorrectly assumed that was the idea for the docker url as well. Feel free to close this PR.